client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts

```diff
@@ -658,16 +658,42 @@ const useInvestigationTable = (parameters: useInvestigationTableParameters): use
         } else if (cellKey === TableHeadersNames.priority) {
             classNames.push(classes.priorityTableCell);
         }
-        if ((isDefaultOrder && !isLoading) &&
-            (rows.length - 1 !== rowIndex) &&
-            rows[rowIndex]?.coronaTestDate &&
-            (getFormattedDate(rows[rowIndex]?.coronaTestDate) !== getFormattedDate(rows[rowIndex + 1]?.coronaTestDate))) {
+
+        if(isLastNestedCell === undefined) {
+            if(cellNeedsABorder(rowIndex)) {
+                classNames.push(classes.rowBorder)
+            }
+            if(isGroupShown){
+                    classNames.push(classes.nestedTableCell);
```

Comment:
too many indentations

Reply:
why? it's not an if-else statement , a cell answer both of these conditions

Reply:
whoops , fixed [here](489f80d)